CONTRIBUTING.rst

```diff
@@ -93,6 +93,23 @@ Contribution Guidelines
 .. _formatting: https://molecule.readthedocs.io/en/latest/testing.html#formatting
 .. _linting: https://molecule.readthedocs.io/en/latest/testing.html#linting
 
+Code Of Conduct
+===============
+
+Please see our `Code of Conduct`_ document.
+
+.. _Code of Conduct: ./.github/CODE_OF_CONDUCT.md
+
+Pull Request Life Cycle and Governance
+======================================
+
+* If your PRs get stuck `join us on IRC`_ or add to the `working group agenda`_.
+* Coding standard is what is enforced by CI, everything else is off topic.
```

Comment:
```suggestion
* The code style standard is what is enforced by CI, everything else is off topic.
```
"Coding" is too generic. Sometimes, when the linter is okay with code, it can still be nonsense: enforced code style is almost irrelevant when talking about quality.